ios/RCTCardIOModule.m

```diff
@@ -122,11 +122,29 @@ - (void)userDidCancelPaymentViewController:(CardIOPaymentViewController *)scanVi
     _reject(@"user_cancelled", @"The user cancelled", nil);
 }
 
+- (NSString *)documentsPathForFileName:(NSString *)name
+{
+    NSArray *paths = NSSearchPathForDirectoriesInDomains(NSDocumentDirectory,NSUserDomainMask, YES);
+    NSString *documentsPath = [paths objectAtIndex:0];
+    
+    return [documentsPath stringByAppendingPathComponent:name];
+}
+
+- (NSString *)saveScannedImageToDocumentsDirectory:(UIImage *)scannedImage {
+    NSData* imageData = UIImagePNGRepresentation(scannedImage);
+    NSString *filePath = [self documentsPathForFileName:@"scannedImage.png"]; //Add the file name
```

Comment:
the file name doesn't seem to be unique, is it meant to be a temporary location?